.eslintrc.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
```

Comment:
There's already a .eslintrc file at the root of the project, can u check if it works for you?

Reply:
Yes, it worked when I reimported it. Thanks!